app/views/warehouse_reports/cas/non_hmis_clients/_client.haml

```diff
@@ -0,0 +1,24 @@
+- candidate = client.last
+%tr
+  %td= candidate[:first_name]&.capitalize
+  %td= candidate[:last_name]&.capitalize
```

Comment:
`capitalize` feels wrong here.  Can we store the original somewhere for display?  I'm worried the McDonalds of the world or Gibson-Aikens will not make sense.